Marlin/ultralcd.cpp

```diff
@@ -3572,7 +3572,8 @@ void kill_screen(const char* lcd_msg) {
       #if EXTRUDERS > 1
         MENU_ITEM_EDIT(float52, MSG_CONTROL_RETRACT_RECOVER_SWAP, &swap_retract_recover_length, -100, 100);
       #endif
-      MENU_ITEM_EDIT(float3, MSG_CONTROL_RETRACT_RECOVERF, &retract_recover_feedrate_mm_s, 1, 999);
+      MENU_ITEM_EDIT(float3, MSG_CONTROL_RETRACT_RECOVERF, &retract_recover_feedrate_mm_s, 1, 999);.
+      MENU_ITEM_EDIT(float3, MSG_CONTROL_RETRACT_SWAPF, &swap_retract_recover_feedrate_mm_s, 1, 999);
       END_MENU();
     }
 
```

Comment:
Needed to make the perfect recover when the nozzle restart during printing 
This option is simply Essential 👍